### PR TITLE
Names for send/request messages in group chat (#1396); refactoring of command sending

### LINKED ISSUE #1396: Requesting message in group chat doesn't contain recipient name. All members see it as request to them [send]
### Description

*Type*: Bug

*Summary*: Requesting message in group chat doesn't contain recipient name. There is no way to see to whom this message is sent, so any user can see it as it was sent to him. We need some indication in the message to show the recipient. In the image below, recipient and another member of the group chat see identical message and both can use it to send ether:
img_1914

#### Expected behavior
Requesting message in the group chat specifies recipient username
#### Actual behavior
Requesting message in the group chat looks similar to all members and all of them can use it to send ether.

### Reproduction
you need to have group chat with 3 mebers
- Open Status
- as member1: request from member2 some ether in group chat
- as member2: check how requesting message looks like and use it to send ether to member1 (it's expected that only member2 can use it or that message shows who is intended recipient - member2)

### Additional Information
* Status version: 0.9.8-56-gd743691+
* Operating System: Android and iOS

### PR DESCRIPTION
**Pull request by alwx**
I've changed the way command's structure look like (for instance, there is no more `:command-message`) and fixes #1396
Probably it also solves other issues.

### CODE REVIEW

**Comment by rasom** [comment]
@alwx please resolve conflicts

File: src/status_im/utils/handlers.cljs [hunk 1]
```diff
@@ -35,7 +35,8 @@
 (defn register-handler
   ([name handler] (register-handler name nil handler))
   ([name middleware handler]
-   (re-core/register-handler name [debug-handlers-names (when js/goog.DEBUG check-spec) middleware] handler)))
+    ;; TODO: enable specs
+   (re-core/register-handler name [debug-handlers-names #_(when js/goog.DEBUG check-spec) middleware] handler)))
```
Comment (flexsurfer) [inline src/status_im/utils/handlers.cljs]: do you have plans to enable specs? :)

**Comment by annadanchenko** [comment]
tested basic features on iOS and Android, looks OK to merge. Will continue testing on develop build

File: src/status_im/utils/handlers.cljs (hunk 1 above)
Comment (alwx) [inline src/status_im/utils/handlers.cljs]: :)

**Pull request merged**